src/main/resources/com/mysql/cj/LocalizedErrorMessages.properties

```diff
@@ -47,6 +47,10 @@ AuthenticationProvider.BadDisabledAuthenticationPlugin=Can''t disable the defaul
 AuthenticationProvider.AuthenticationPluginRequiresSSL=SSL connection required for plugin ''{0}''. Check if "useSSL" is set to "true".
 AuthenticationProvider.UnexpectedAuthenticationApproval=Unexpected authentication approval: ''{0}'' plugin did not reported "done" state but server has approved connection.
 
+AuthenticationAwsIamPlugin.UnsupportedHostname=Unsupported AWS hostname ''{0}''.
```

Comment:
The message probably needs to be a bit more descriptive on what's wrong with host name. I'd add something like "Amazon domain name in format *.rds.amazonaws.com is expected."